### DISCUSSION TITLE
How can I pin app to taskbar??

### DISCUSSION THREAD

**Discussion by michaelmuselik** [valinet/ExplorerPatcher#1603] [Q&A]
I cannot pin any app on W10 taskbar. What am I doing wrong?
![2023-06-30_07h46_51](https://github.com/valinet/ExplorerPatcher/assets/73330785/0acd34b7-a130-4d45-961a-ecbc74cd6bbe)

**Comment by pyrates999** [comment]
Duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1108

**Comment by vippium** [comment]
> I cannot pin any app on W10 taskbar. What am I doing wrong? ![2023-06-30_07h46_51](https://user-images.githubusercontent.com/73330785/250005946-0acd34b7-a130-4d45-961a-ecbc74cd6bbe.png)

#1591
